### PR TITLE
Increase spacing between number of people joined and icon

### PR DESCRIPTION
**Pull request by lucyxubroad**
Increase spacing between number of people joined and icon
• add edge insets to content and image of people icon to increase spacing between number of people joined and icon.

### CODE REVIEW

**Comment by lucyxubroad** [comment]
**Attached screenshot:
** screen shot 2019-02-16 at 1 29 07 am

File: Clicker.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -902,7 +902,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n#check if env-vars.sh exists\nif [ -f ./env-vars.sh ]\nthen\nsource ./env-vars.sh\nfi\n$PODS_ROOT/Sourcery/bin/sourcery --sources . --templates Sourcery/keys.stencil --output Secrets --args apiURL=$API_URL,apiDevURL=$API_DEV_URL,fabricAPIKey=$FABRIC_API_KEY\necho $API_URL\necho \"created keys.generated.swift file\"\n";
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n#check if env-vars.sh exists\nif [ -f ./env-vars.sh ]\nthen\nsource ./env-vars.sh\nfi\n$PODS_ROOT/Sourcery/bin/sourcery --sources . --templates Sourcery/keys.stencil --output Secrets --args random=\"yeet\",apiURL=$API_URL,apiDevURL=$API_DEV_URL,fabricAPIKey=$FABRIC_API_KEY\necho \"created keys.generated.swift file\"\n";
```
Comment (mindylou) [inline Clicker.xcodeproj/project.pbxproj]: lol can you change it from yeet to pollo `
```suggestion
			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n#check if env-vars.sh exists\nif [ -f ./env-vars.sh ]\nthen\nsource ./env-vars.sh\nfi\n$PODS_ROOT/Sourcery/bin/sourcery --sources . --templates Sourcery/keys.stencil --output Secrets --args random=\"pollo\",apiURL=$API_URL,apiDevURL=$API_DEV_URL,fabricAPIKey=$FABRIC_API_KEY\necho \"created keys.generated.swift file\"\n";
```

Comment (kevinchan159) [reply Clicker.xcodeproj/project.pbxproj]: Y33T

Comment (lucyxubroad) [reply Clicker.xcodeproj/project.pbxproj]: Resolved with latest push!

**Pull request merged**